A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -83,6 +83,9 @@ var A320_Neo_UpperECAM;
         get templateID() {
             return "UpperECAMTemplate";
         }
+        static refresh() {
+            console.log("REFRESH");
```

Comment:
This looks like forgotten debug logging.

Reply:
Pushed - Remove refresh function